xml/admin_ceph_datamgm.xml

```diff
@@ -67,31 +70,36 @@
  <para>
   A custom &crushmap; can also help you identify the physical locations where
   &ceph; stores redundant copies of data when the placement group(s) (refer to
-  <xref linkend="op-pgs"/>) associated with a failed host are in a degraded
-  state.
+  <xref
```

Comment:
Not sure that splitting xref across 3 lines is useful :-)

Reply:
That was daps-xmlformat, I fear. :(